### ISSUE TITLE
Browsing history and Live bookmarks/unresponsive Firefox

### ISSUE THREAD

**Issue by cfinke** [cfinke/RSS-Ticker#10]
<b>What steps will reproduce the problem?</b>
1. When clearing the browsing history (needed for privacy - no compromise)
   the read feeds are also cleared. This is when doing a manual clear or when
   using the option to clear it when Firefox closes.

<b>What is the expected output? What do you see instead?</b>

After clearing the browsing history and restarting Firefox, I want to have
only new items from feeds displayed. Instead all items (including previous)
from all feeds are displayed again.

<b>What version of the product are you using? On what operating system?</b>

Version 2.0.3 on Windows XP Pro, service pack 2 as well as on Ubuntu 9.04. 

<b>Please provide any additional information below.</b>

Couldn't a database be kept in the plugin's folder? Possibly one text file
(a .csv file would suffice) for each feed. This could be used to add a
status flag to each item. Items older than a specific amount of days
shouldn't be shown IMO (I recommend 30 days), and this fact could be used
to clear old items from the database.

Also, the fact that RSS Ticker &quot;feeds&quot; off the live bookmarks is probably
the root of this problem. Keeping a separate database would mean not having
to use them, which by itself would also speed up Firefox, as there has been
some bug reports related to them and an unresponsive Firefox.

In other words, using a database would solve two problems.

---

Issue imported from Google Code
Author: [george.michelle.butler](http://code.google.com/u/george.michelle.butler/)
Published: 2009-07-26T21:09:29.000Z
Link: http://code.google.com/feeds/issues/p/rss-ticker/issues/10/comments/full
Status: New

**Comment by cfinke** [comment]
I also experience similar issues while starting firefox.

Xavier

---

Comment imported from Google Code
Author: [msgxavier](http://code.google.com/u/msgxavier/)
Published: 2009-07-26T22:56:32.000Z
Link: http://code.google.com/p/rss-ticker/issues/detail?id=10#c1
